providers/gpgpu/bin/gpu_passthrough.py

```diff
@@ -134,6 +155,32 @@ def test_lxdvm_gpu(args):
         logging.info("Waiting for %s to be up", instance.name)
         instance.wait_until_running()
 
+        instance.stop(force=True)
+
+        if args.qemuopts:
+            logging.info(
+                "Setting user-provided QEMU options: %s", args.qemuopts
+            )
+            instance.set_config("raw.qemu={}".format(args.qemuopts))
+
+        if int(args.vmcpus) > 0:
```

Comment:
Let's use Python's truthy values:

```suggestion
        if args.vmcpus:
```